pkg/util/version/version.go

```diff
@@ -36,6 +36,14 @@ func ParseVersion(version string) Version {
 	return Version{Major: major, Minor: minor}
 }
 
+func ParseVersions(versions string) []Version {
+	var result []Version
+	for _, version := range strings.Split(versions, " ") {
+		result = append(result, ParseVersion(version))
+	}
+	return result
+}
+
```

Comment:
It looks like this function is not used anywhere, so it should be removed.

Reply:
Done